app/views/plans/new.html.erb

```diff
@@ -50,7 +50,7 @@
         <div class="form-group col-md-6">
            <em class="sr-only"><%= research_org_tooltip %></em>
            <% dflt = @orgs.include?(current_user.org) ? current_user.org : nil %>
-           <%= fields_for :org, @plan.org do |org_fields| %>
+           <%= f.fields_for :org, @plan.org do |org_fields| %>
```

Comment:
I stupidly missed this when doing the strong params. `f.` ensures that the field id is `plan_org_org_name` instead of `org_org_name` 🙄